go.mod

```diff
@@ -175,6 +174,7 @@ require (
 	github.com/gobuffalo/flect v1.0.2 // indirect
 	github.com/godbus/dbus/v5 v5.1.0 // indirect
 	github.com/golang-jwt/jwt/v4 v4.5.0 // indirect
+	github.com/golang/protobuf v1.5.3 // indirect
```

Comment:
Sorry, I am not very experienced in Go. How does version number work here, is it auto-generated? I see that the https://github.com/protocolbuffers/protobuf-go/blob/master/go.mod requires `v1.5.0`

Reply:
Go uses minimal version selection. See notably [this Go ref](https://go.dev/ref/mod#minimal-version-selection) and [this blog post](https://research.swtch.com/vgo-mvs) by Russ Cox.